frontend/src/index.js

```diff
@@ -6,13 +6,15 @@ import './index.css';
 import App from './App';
 import Footer from './components/Footer/index';
 import Benefits from './components/Benefits/index';
+import OrderListComponent from './components/OrderList';
 
 ReactDOM.render(
   <BrowserRouter>
     <Switch>
       <Route path="/" exact={true} component={App} />
       <Route path="./contato" component={Footer} />
```

Comment:
Não relacionado com essa PR, mas esse `.` no início dessa rota tá esquisito. Sabem se isso funciona corretamente? :thinking:

Reply:
Pois é, ele ta em todas as rotas pelo que vejo